### ISSUE TITLE
Question: Why is the evaluation result dropped from one stage to the next?

### ISSUE THREAD

**Issue by johannes-b** [keptn/keptn#6167]
## Use case

As a user, I have an `approval` task set as first task in my delivery sequence of production: 

```
apiVersion: "spec.keptn.sh/0.2.2"
kind: "Shipyard"
metadata:
  name: "shipyard-sockshop"
spec:
  stages:

    - name: "staging"
      sequences:
        - name: "delivery"
          triggeredOn:
            - event: "dev.delivery.finished"
          tasks:
            - name: "deployment"
              properties:
                deploymentstrategy: "blue_green_service"
            - name: "test"
              properties:
                teststrategy: "performance"
            - name: "evaluation"
            - name: "release"
            - name: "approval"
              properties:
                pass: "manual"
                warning: "manual"

    - name: "production"
      sequences:
        - name: "delivery"
          triggeredOn:
            - event: "staging.delivery.finished"
          tasks:
            - name: "approval"
              properties:
                pass: "manual"
                warning: "manual"
            - name: "deployment"
              properties:
                deploymentstrategy: "blue_green_service"
            - name: "release"
```

I'm used to the behavior that the approval task shows the last evaluation result: 

![image](https://user-images.githubusercontent.com/729071/143261038-b75e4f13-bdb7-4b4c-aaeb-cd9ecc89e2bd.png)

However, when using the `approval` task as the first task, the evaluation score is not shown. 

![image](https://user-images.githubusercontent.com/729071/143261857-08041657-79a8-4d92-9d9b-90a7f8e097a3.png)

### Investigation

I figured out that during the promotion from staging to production, the `score:100` dropped to `0`. Why is this done? 

**In staging:**
![image](https://user-images.githubusercontent.com/729071/143261471-79bb8f94-5e5e-4ac9-8143-484e1b036b03.png)

**In production:**
![image](https://user-images.githubusercontent.com/729071/143261726-b6b24705-8091-45d2-b46a-89f728c26ef2.png)

**Comment by Dynatrace-Adam-Gardner** [comment]
Move to [discussions](https://github.com/keptn/keptn/discussions)?

**Comment by RealAnna** [comment]
![image](https://user-images.githubusercontent.com/89971034/145999263-2abdbaa9-023d-46c3-bdc4-8098ac598b17.png)
I tested the same shipyard on my cluster running 0.11.3 and the score is correctly propagated. I believe the issue has been solved here  #5691
